### PR TITLE
Remove unused audio playback support

### PR DESCRIPTION
**Pull request by mistydemeo**

### CODE REVIEW

**Comment by Hwesta** [comment]
Also remove the Sounds table/model and foreign keys to it?

**Comment by mistydemeo** [comment]
Wow, I didn't even know we'd made a model for it.

File: src/MCPServer/lib/plantUMLTextGenerator/main.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def jobChainLinkExitCodesTextGet(indent, exitCode, nextMicroServiceChainLink, ex
...
+    sql = """SELECT MicroServiceChainLinks.currentTask, MicroServiceChainLinks.defaultNextChainLink, TasksConfigs.taskType, TasksConfigs.taskTypePKReference, TasksConfigs.description, MicroServiceChainLinks.reloadFileList, Sounds.fileLocation, MicroServiceChainLinks.defaultExitMessage, MicroServiceChainLinks.microserviceGroup, StandardTasksConfigs.execute FROM MicroServiceChainLinks JOIN TasksConfigs on MicroServiceChainLinks.currentTask = TasksConfigs.pk LEFT OUTER JOIN StandardTasksConfigs ON TasksConfigs.taskTypePKReference = StandardTasksConfigs.pk WHERE MicroServiceChainLinks.pk = %s;"""
```
Comment (Hwesta) [inline src/MCPServer/lib/plantUMLTextGenerator/main.py]: Drop `Sounds.fileLocation` from the fields list

**Comment by Hwesta** [comment]
Yeah - I added it so when we start creating DB things from the models everything would still work.

**Comment by mistydemeo** [comment]
It looks like the plantUML generator is not working, and I don't think we've used it in a long time, so I'm tempted to drop it entirely in another commit.

**Comment by Hwesta** [comment]
Sounds good to me

**Pull request merged**